src/sentry/api/endpoints/project_backfill_similar_issues_embeddings_records.py

```diff
@@ -33,15 +33,21 @@ def post(self, request: Request, project) -> Response:
 
         last_processed_id = None
         only_delete = False
+        enable_ingestion = True
```

Comment:
is there a good reason to default this to `True`? May be less risky to default false

Reply:
I figured we'd typically want to have it be true, but makes sense either way

Reply:
yeah unless it greatly simplifies things id prefer we default to false so we can make it more explicit whether or not its happening when we do the backfills

Reply:
(worst case for example we mean to do a test backfill of a random language and brick their ingestion bc we forget to pass the flag)